arlas-rest/src/main/java/io/arlas/server/rest/explore/countDistinct/CountDistinctRESTService.java

```diff
@@ -0,0 +1,213 @@
+/*
+ * Licensed to Gisaïa under one or more contributor
+ * license agreements. See the NOTICE.txt file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Gisaïa licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.arlas.server.rest.explore.countDistinct;
+
+import com.codahale.metrics.annotation.Timed;
+import io.arlas.server.app.Documentation;
+import io.arlas.server.exceptions.ArlasException;
+import io.arlas.server.model.CollectionReference;
+import io.arlas.server.model.request.CountDistinct;
+import io.arlas.server.model.request.MixedRequest;
+import io.arlas.server.model.response.CountDistinctResponse;
+import io.arlas.server.model.response.Error;
+import io.arlas.server.rest.explore.ExploreRESTServices;
+import io.arlas.server.services.ExploreServices;
+import io.arlas.server.utils.CheckParams;
+import io.arlas.server.utils.ParamsParser;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+public class CountDistinctRESTService extends ExploreRESTServices {
+    public CountDistinctRESTService(ExploreServices exploreServices) {
+        super(exploreServices);
+    }
+
+    @Timed
+    @Path("{collection}/_countDistinct")
+    @GET
+    @Produces(UTF8JSON)
+    @Consumes(UTF8JSON)
+    @ApiOperation(value = "Count distinct", produces = UTF8JSON, notes = "Counts the approximate distinct values of a given field, given the filters", consumes = UTF8JSON, response = CountDistinctResponse.class)
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Successful operation", response = CountDistinctResponse.class, responseContainer = "CountDistinctResponse"),
```

Comment:
I am not sure that this responseContainer is valid, see http://docs.swagger.io/swagger-core/apidocs/com/wordnik/swagger/annotations/ApiOperation.html#responseContainer() if it applies here

Reply:
The response is not a list, so I removed the responseContainer